internal/cli/root/mongocli/builder.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package root
+package mongocli
```

Comment:
[non-blocking] I think I would do this `mongocliroot` instead of having to sub packages inside of root but that's just me